ZipBuilder/Sources/sem-versions/PackageReader/CocoapodsReader.swift

```diff
@@ -0,0 +1,141 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import Foundation
+
+import ShellUtils
+
+import PathKit
+
+class CocoapodsReader: PackageReader {
```

Comment:
Nit: please add a bit of documentation as to what this class does, along with at least the public methods. Even a single line with `///` at the start is good.